modules/nf-core/samtools/bgzip/main.nf

```diff
@@ -0,0 +1,60 @@
+process SAMTOOLS_BGZIP {
+    tag "$fasta"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/samtools:1.21--h50ea8bc_0' :
+        'biocontainers/samtools:1.21--h50ea8bc_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("${output}")      , emit: fa
+    path "versions.yml"                     , emit: versions
```

Comment:
why `fa` and not `fasta`?

Reply:
I'd rather we be consistent with emit names:
cf
https://github.com/nf-core/modules/blob/master/modules/nf-core/custom/catadditionalfasta/main.nf

Reply:
also while I'm being picky, why sooo much space before the commas?

Reply:
Almost ready to push a change (re-running lints and checks)

For `fa` maybe I was worried that I was not allowed to reuse an input name.  And the spaces are leftovers from previous iterations where I changed only one line and did not want to have a space-only change in the other line.